app/repositories/users_repository.rb

```
@@ -8,7 +8,7 @@ def active
   end
 
   def find_by_id(id)
-    active.find(id)
+    all.find(id)
```

Doesn't this change make this method redundant? (It basically does nothing now except passing function param).